app/code/community/Snowdog/CreditagricoleRaty/sql/snowcreditagricoleraty_setup/mysql4-upgrade-1.5.3-1.5.4.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+/* @var $installer Mage_Core_Model_Resource_Setup */
+$installer = $this;
+$installer->startSetup();
+
+$installer->run(
+    "ALTER TABLE {$this->getTable('snowcreditagricoleraty/application')}
+    MODIFY incremental_id VARCHAR(50)"
```

Comment:
The previous definition contains `NOT NULL`, so probably this is missing here. In addition, I don't see a key/index for this column, so it would be nice to add it.